js/stronghold.js

```diff
@@ -506,6 +515,7 @@ module.exports = class stronghold extends Exchange {
 
     async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
         await this.loadMarkets ();
+        const market = this.market (symbol);
```

Comment:
If `symbol` is undefined, the signature says that it's optional, so, we have to do this:

```JavaScript
async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
        await this.loadMarkets ();
        let market = undefined;
        if (symbol !== undefined) {
            market = this.market (symbol);
        }
        // ...
        return this.parseOrders (..., market, ..., ...);
}
```

or this:

```JavaScript
async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
        await this.loadMarkets ();
        if (symbol === undefined) {
            throw new ArgumentsRequired (this.id + ' fetchClosedOrders requires a symbol argument');
        }
        const market = this.market (symbol);
        // ...
        return this.parseOrders (..., market, ..., ...);
}
```